policies/policies.go

```diff
@@ -0,0 +1,301 @@
+//  Copyright 2018 Google Inc. All Rights Reserved.
```

Comment:
2019

Reply:
This file stayed largely the same, just name change so we want to keep the contents under the old copyright